toplevel/toploop.ml

```diff
@@ -140,10 +139,25 @@ let parse_mod_use_file name lb =
        ]
    ]
 
-(* Hooks for initialization *)
+(* Hook for initialization *)
```

Comment:
All of this needs to go to opttoploop as well

Reply:
Indeed, I'll wait until this is approved and then sync the native toplevel in order to cut down the amount of intermediate work.